osu.Game/Screens/Play/GameplayMenuOverlay.cs

```diff
@@ -142,6 +140,8 @@ private void load(OsuColour colours)
                 },
             };
 
+            State.ValueChanged += s => InternalButtons.Deselect();
```

Comment:
Any reason for moving into here instead of `LoadComplete()`?

Reply:
Just for safety... `VisibilityContainer` still has very weird state handling logic that I was preferring not to drill into (I know it does some kind of state change handling/allowance for `InitialState` changing in BDL or similar...)

Reply:
Mmmk. I'm 95% expecting this to fail in some edge case with some component trying to make the overlay visible and it dying on a transform-from-non-load-thread exception. But for now I'll :see_no_evil:

Reply:
Well it's worked until now (such a case would have already been hitting a nullref). We can move it if you want, but please test at your end if so 😅